src/errors.rs

```diff
@@ -0,0 +1,25 @@
+pub type Result<T = ()> = std::result::Result<T, Error>;
+
+#[derive(Debug, thiserror::Error)]
+pub enum Error {
+    #[error("{0}")]
+    FromUtf8(#[from] std::string::FromUtf8Error),
+    #[error("{0}")]
+    InvalidIp(#[from] std::net::AddrParseError),
+    #[error("{}", .0.str)]
+    Lxc(Lxc),
```

Comment:
Use `LXCContainer` instead of `Lxc` as your're referring to an error happening within the context of an LXC container, and not to any errors reported by `liblxc`.